### ISSUE TITLE
Syntax error when running latest image version 0.2.65

### ISSUE THREAD

**Issue by devtobi** [mayope/keycloakmigration#82]
With the newest version `0.2.65` I get the following error when executing:
```
./entrypoint.sh: line 13: syntax error: unexpected end of file (expecting "fi")
```

Prior to this version I used `0.2.55` which worked perfectly fine.

**Comment by klg71** [comment]
Hey @devtobi  Thanks for opening this issue. I will investigate it.

**Comment by klg71** [comment]
I released docker image 0.2.67 it should start again. I released the entrypoint.sh with CRLF :/

**Comment by devtobi** [comment]
Thank you very much for investigating. I just tried it out and it works like a charm again. :) Thank you for that quick fix. :)
I think you should add documentation or create a new GitHub release to indicate the Docker image 0.2.65 has this issue and 0.2.67 fixes it.
